src/index.js

```diff
@@ -0,0 +1,64 @@
+// Магазин
+
+import { Client } from './Client.js';
+import { Product } from './Product.js';
+import { Shop } from './Shop.js';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Varlyle/my-first-project/pullRequest?prid=7306919)
```suggestion
import { Shop } from "./Shop.js";
```